source/index.html.md

```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
 | organisation_id  | ID of the organisation where the animal belongs to. | `required`       |
 | include_archived | Include archived animals.                           | default: `false` |
 
+## Put Animals
+
+> Request
+
+```python
+import requests
+
+endpoint = 'https://api-staging.smaxtec.com/integration/v2'
+organisation_id = '123456qwertz'
+optional_parameters = '?include_archived=false'
+route = endpoint + '/organisations/' + organisation_id + '/animals' + optional_parameters
+token = 'yx2zvuB8JD8ppwGti84OT8Muq5eiB2b2EZqsqC-HOXUvLSg'
+user_agent = 'HerdManagerSoftwareXYZ/2.4'
+headers = {
+    'accept': 'application/json',
+    'User-Agent': user_agent,
+    'Authorization': 'bearer ' + token
+}
+
+r = requests.put(route, headers=headers)
+
+status_code = r.status_code
+all_animals = r.json()
+```
+
+```bash
+curl -X GET "[endpoint]/organisations/[organisation_id]/animals" \
+        -H  "accept: application/json" \
+        -H  "Authorization: bearer [token]" \
+        -H  "User-Agent: [user_agent]"
+```
+
+> Response example
+
+```json
+[
+    {
+        "birthday": null,
+        "sensor": null,
+        "organisation_id": "618cfada71de04ff6bb72216",
+        "official_id_rule": "AT",
+        "current_device_id": null,
+        "mark": "AT111111119",
+        "_id": "618cfada352092fb27a899c9",
+        "race": null,
+        "official_id": "AT111111119",
+        "archived": false,
+        "tags": [],
+        "display_name": "AT111111119",
+        "last_readout": null,
+        "active": false,
+        "mixed_race": null,
+        "created_at": "2021-11-11T11:13:30.855351",
+        "lactation_status": "Young_Cow",
+        "location": null,
+        "name": "AT111111119",
+        "group_id": null
+    },
+    {
+        "birthday": null,
+        "sensor": null,
+        "organisation_id": "618cfada71de04ff6bb72216",
+        "official_id_rule": "AT",
+        "current_device_id": null,
+        "mark": "AT111111118",
+        "_id": "618cfadb86dcfb58c33bb639",
+        "race": null,
+        "official_id": "AT111111118",
+        "archived": false,
+        "tags": [],
+        "display_name": "AT111111118",
+        "last_readout": null,
+        "active": false,
+        "mixed_race": null,
+        "created_at": "2021-11-11T11:13:31",
+        "lactation_status": "Young_Cow",
+        "location": null,
+        "name": "AT111111118",
+        "group_id": null
+    }
+]
+```
+
+This endpoint add multiple animals for a provided `organisation_id`.
+
+**HTTP Request**
+
+`PUT "[endpoint]/organisations/[organisation_id]/animals"`
+
+**URL Parameters**
+
+| Parameter        | Description                                         |                  |
+| ---------------- | --------------------------------------------------- | ---------------- |
+| organisation_id  | ID of the organisation where the animal belongs to. | `required`       |
+| include_archived | Include archived animals.                           | default: `false` |
```

Comment:
the `include_archived` isn't an accepted parameter. this needs to be deleted.

instead the animals should be put here. you can reference to the parameters from **Create/Update Single Animal**